### DISCUSSION TITLE
Request code support for the plugin , Geyser

### DISCUSSION THREAD

**Discussion by Tsoccerguy3** [EssentialsX/Essentials#4594] [Ideas and Feature Suggestions] [answered]
Request support for the plugin , Geyser

The Geyser DEV mentioned that Essential X needs to support Mojangs API and be compiled with the API responsible for the Auto complete feature . The code to support it , must be on the plugin side. Since Geyser is a extension to  the multi - player support that allows all Bedrock Devices to play on a Java server  and is a new feature , plugins DEV are not aware it exits. 
Please support cross platform , 

https://github.com/GeyserMC/Geyser

**Comment by JRoy** [comment, marked as answer]
If the geyser devs want to PR a specific change, I'll review it; But I'm not going to make any major changes myself to support their platform.

**Reply by Proximyst** [reply]
Thank you, @JRoy!

**Reply by Tsoccerguy3** [reply]
If you review their project , you will see that they are doing what Mojang should have done in the first place , allow all devices and versions play on the same server. I have tested logon to Paper 1.17.1 from Bedrock clients on the following Devices , Xbox , Nintendo switch  , Apple phone , Android phone and playstation versions of Minecraft . Essential X commands are working . The problem is that EssX is not supplying the auto completion of commands such as /warp . I'm guessing that EssX would need to add in some more support to Mojangs API mappings. I saw the Mojang API's somewhere on the GIT

**Reply by JRoy** [reply]
Thanks for your interest
